test/minify

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Make sure the webserver process dies when we exit
+trap 'kill $(jobs -p)' EXIT
+
+# Create an asset bundle the recommended way
+APOS_BUNDLE=1 node app apostrophe:generation || exit 1
+# Fire up the site using that bundle's assets
+APOS_BUNDLE=1 node app &
+sleep 10
+# Verify we can get the home page
+wget http://localhost:3000 -O /tmp/$$.html || exit 1
+# http://localhost:3000/uploads/assets/ck199jeiq00013quk6zgkmbd2/apos-minified/anon-ck199jeiq00013quk6zgkmbd2.css
```

Comment:
Stray comment here.

Reply:
I left that there so I could see what I was trying to detect.

Reply:
Got it. Using a `NOTE: ` prefix with a few words of explanation might help those who come along later.